import time



def point_right():
    safe_position = {
                        'head'          :  self.origin["head"],
                        'right_shoulder':  120,
                        'right_elbow'   :  self.origin["right_elbow"],
                        'left_shoulder' :  800,
                        #'left_elbow':     self.origin["left_elbow"]
                    }
